--- decompiled_assembly/Assembly-CSharp/CaravanPanel.cs	2024-08-25 01:50:00.296210000 +0300
+++ work/Assembly-CSharp/CaravanPanel.cs	2024-08-25 01:50:00.845210200 +0300
@@ -33,7 +33,7 @@
 
 	private void UpdateUpgradeCargoButton()
 	{
-		this.increaseCapacityText.text = LocalizationManager.instance.GetLocalizedValue("carg2") + Caravan.instance.quantityCargoCapacityIncrease;
+		this.increaseCapacityText.text = LocalizationManager.instance.GetLocalizedValue("carg2") + Caravan.instance.quantityCargoCapacityIncrease.ToString();
 		this.costText.text = Caravan.instance.costCargoCapacityIncrease.ToString("0");
 	}
 
@@ -64,13 +64,7 @@
 	private void Update()
 	{
 		UIControl.instance.panEnabled = false;
-		this.cargoUsedText.text = string.Concat(new object[]
-		{
-			this.cargoString,
-			Caravan.instance.cargoFilled,
-			"/",
-			Caravan.instance.cargoCapacity
-		});
+		this.cargoUsedText.text = this.cargoString + Caravan.instance.cargoFilled.ToString() + "/" + Caravan.instance.cargoCapacity.ToString();
 		if (Caravan.instance.cargoCapacity < Caravan.instance.maxCargoCapacity && (double)Caravan.instance.costCargoCapacityIncrease < CentralAdmin.instance.gold)
 		{
 			this.upgradeCargoButton.interactable = true;
